app/js/controllers/events.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+angular
+.module('app.controllers')
+.controller('EventsCtrl', function($scope, $location, Restangular) {
+	$scope.loadingPromise = Restangular.one('events/next-10-publicly')
```

Comment:
I know this is an API discussion, but I don't know if I like that route. Why did we hardcode a route for this instead of doing something like `/events?sort=startDateTime&page=0&pageSize=10`?